docs/architecture-and-design.md

```diff
@@ -386,6 +390,20 @@ Most [Intents](#intents) are easy to break down into  [Primitives](#primitives),
 
 In order for a robot to move to the desired destination of a `MoveIntent`, the Navigator will use various path-planning algorithms to find a path across the field that does not collide with any robots or violate any restrictions set on the `MoveIntent`. The Navigator then translates this path into a series of `MovePrimitives`, which are sent to the robot sequentially so that it follows the planned path across the field.
 
+### Path Manager
+The `Path Manager` is responsible for generating a set of paths that don't collide. It is given a set of [Path Objective](#path-objective)s and [Path Planner](#path-planner), and it will generate paths using the given path planner and arbitrate between paths to prevent collisions. One approach is to add extra obstacles to cause the path planner to generate paths that avoid potential collisions; this is what `VelocityObstaclePathManager` implements. The approach is somewhat like using [Minkowski space](https://en.wikipedia.org/wiki/Minkowski_space), but where we assume that a robot will occupy all the positions along the path for the next time step. However, it's a very crude implementation.
+
+### Path Objective
+A path objective is used to communicate between the navigator and the path manager. It conveys information for generating one path, such as start, destination, and obstacles.
```

Comment:
```suggestion
A path objective is a simple datastructure used to communicate between the navigator and the path manager. It conveys information for generating one path, such as start, destination, and obstacles. Path Objectives use very simple datastructures so that Path Planners do not need to know about any world-specific datastructures, such as Robots or the Field.
```

suggestion for adding a bit more detail? (Will need to link things properly)